AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/resources/ResourceTypesRest.java

```diff
@@ -71,4 +83,16 @@ public List<ResourceTypeDTO> getRootResourceTypes() {
                 .map(ResourceTypeDTO::new)
                 .collect(Collectors.toList());
     }
+
+    @DELETE
+    @Path("/{id : \\d+}")
+    @ApiOperation(value = "Delete a resource type")
+    public Response deleteResourceType(@PathParam("id") Integer id) throws NotFoundException {
+        try {
+            resourceTypeDomainService.removeResourceType(id);
```

Comment:
Currently, it's possible to delete both predefined and root resource types through the API. Since you never want to delete predefined resource types, you could add some validation that only allows deletion of root resource types?